lib/Sema/CSDiagnostics.cpp

```diff
@@ -474,7 +474,7 @@ bool MissingConformanceFailure::diagnoseTypeCannotConform(
   }
 
   emitDiagnostic(diag::type_cannot_conform,
-                 nonConformingType->isExistentialType(), nonConformingType,
+                 nonConformingType->isExistentialType(), nonConformingType, nonConformingType->isEqual(protocolType),
```

Comment:
Nit: Could you add a line-break here (and similarly hard-wrap lines in other places)? For most things, including code and documentation, we follow the [LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#source-code-width) which uses an 80-column width.

There is a tool `llvm-project/clang/tools/clang-format/clang-format-diff.py` in case you'd like to use it to format diffs, but that sometimes does too much.

Reply:
Okay, thanks @varungandhi-apple, I will make the necessary changes.